CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # 2.0.0
 
-* Bumped Ion Go to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
+* Bumped Ion Go dependency to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
```

Comment:
can we add the Enhancements header like we did for breaking changes header down below?

```
## :tada: Enhancements
* Bumped Ion Go dependency to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
```

Reply:
Done.

Reply:
Missing `:tada:` infront of Enhancements

Reply:
Done.